config.yaml

```diff
@@ -1,6 +1,6 @@
 default:
  log_file: "/var/log/host-os/builds.log"
- repositories_path: "/var/lib/host-os/repositories"
+ packages_repos_target_path: "/var/lib/host-os/repositories"
```

Comment:
Ok. This directory name makes sense with the current scope of the directory. If the scope is updated with @gutoyr 's patch, he may rename it.